### PR TITLE
Use @rails/webpacker in place of CRA

### PR DESCRIPTION
**Pull request by eckdanny-osi**
## Description

Adds the `webpacker` gem with the `react` boilerplate for better developer experience (DX).

`create-react-app` (CRA) with `react-scripts-cssmodules` is good in isolation, but significantly painful DX after deeper integration w/ rails app (e.g.; auth). Rails is still the gateway. The change(s) introduced here:

 - Replaces the front-end build routine ~CRA~ -> `@rails/webpacker`
 - No more manual production bundling (hot reloading 👍)
 - data bridge (json) b/n rails and the react app
 - SAME set up as other CWDS digital products

## WiP TODOs

 - [x] blessing from CANS team
 - [ ] verify/mitigate impact in CI tests
 - [ ] verify/mitigate impact in build routines

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
_Hooray!_ Got the `jest` test suite running again:

```
Test Suites: 27 passed, 27 total
Tests:       134 passed, 134 total
Snapshots:   0 total
Time:        6.404s, estimated 9s
Ran all test suites.
✨  Done in 7.10s.
```

**Comment by eckdanny-osi** [comment]
all yours now @jonbwalker

File: app/javascript/Application/styles/global/card.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.card {
+  min-width: 30rem; }
+
+.card-header-cans {
+  background-color: #114161; }
```
Comment (edgibbs) [inline app/javascript/Application/styles/global/card.css]: Nothing major, but should this use one of the colors from the style guide?

File: app/javascript/Application/styles/global/card.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  color: #fff;
+  font-size: 2.0rem; }
+
+.card-header-cans-button {
+  float: left;
+  background-color: transparent; }
+
+.card-header-cans-button > span:first-of-type {
+  padding: 1.5rem 3rem 1.5rem 3rem;
+  background-color: #09798e;
```
Comment (edgibbs) [inline app/javascript/Application/styles/global/card.css]: Same color question here.

File: app/javascript/packs/hello_react.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```
Comment (edgibbs) [inline app/javascript/packs/hello_react.jsx]: Can this be deleted?

File: app/javascript/Application/styles/global/card.css (hunk 1 above)
Comment (jonbwalker) [reply app/javascript/Application/styles/global/card.css]: This file and all other .css files are being removed in this PR (981e7a1). Webpacker allows us to reference the .sass files directly from our js files. (which is really nice).

File: app/javascript/packs/hello_react.jsx (hunk 3 above)
Comment (jonbwalker) [reply app/javascript/packs/hello_react.jsx]: Yep, it's gone now! a24d8de

**Comment by eckdanny-osi** [comment]
@jonbwalker need any help resolving conflicts from #61?

**Pull request merged**